ctk/idp/coverage/NotTested.md

```diff
@@ -0,0 +1,59 @@
+| 2.5.1.6 - Element <ProxyRestriction> | A relying party acting as an asserting party MUST NOT issue an assertion that itself violates the restrictions specified in this condition on the basis of an assertion containing such a condition. | Difficult to test. Would need to intercept interactions between the proxying party and the asserting party.
+| 2.5.1.6 - Element <ProxyRestriction> | A Count value of zero indicates that a relying party MUST NOT issue an assertion to another relying party on the basis of this assertion. | Difficult to test. Would need to intercept interactions between the proxying party and the asserting party.
+| 2.5.1.6 - Element <ProxyRestriction> | If greater than zero, any assertions so issued MUST themselves contain a <ProxyRestriction> element with a Count value of at most one less than this value. | Difficult to test. Would need to intercept interactions between the proxying party and the asserting party.
+| 2.7.3.1 - Element <Attribute> | This attribute's value MUST NOT be used as a basis for formally identifying SAML attributes. | Impossible to verify
+| 2.7.3.1 - Element <Attribute> | Within an \<AttributeStatement>, if the SAML attribute exists but has no values, then the <AttributeValue> element MUST be omitted. | Impossible to know if the attribute has no value from a high-level perspective. Currently the CTK only tests to make sure there is a value within the AttributeValue element. For example: putting an attribute value of "empty" would be interpreted as a valid value by the CTK.
+| 2.7.3.1 - Element <Attribute> | Any other uses of the element by profiles or other specifications MUST define the <Attribute> semantics of specifying or omitting elements. <AttributeValue> | Not verifying custom profiles or their specifications.
+| 2.7.3.1.1 - Element <AttributeValue> | If a SAML attribute includes an empty value, such as the empty string, the corresponding <AttributeValue> element MUST be empty | Impossible to know if an Attribute had an empty string value from a high level perspective, i.e. "empty".
+| 2.7.3.1.1 - Element <AttributeValue> | If a SAML attribute includes a "null" value, the corresponding <AttributeValue> element MUST be empty and MUST contain the reserved xsi:nil XML attribute with a value of "true" or "1". | Impossible to verify if an Attribute had a null value
+| 2.7.3.2 - Element <EncryptedAttribute> | The encrypted content MUST contain an element that has a type of or derived from AttributeType. | It would take a lot of effort to verify the type of the decrypted element and provides little value. However, the extended element's SAML defined base values will be verified by the schema validator.
```

Comment:
Same as the other encrypted elements, it may be reassuring to know the decrypted value is schema validated even if the type is not verified